Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Initialize the OTP builder image
+FROM debian:stable as builder
+
+RUN apt-get update && apt-get install -y --no-install-recommends curl git ca-certificates openssh-client
+
+WORKDIR /java/
+
+# Download the JRE for copying to the image to run the OTP server
+RUN curl -Lo jre17.tar.gz https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.7%2B7/OpenJDK17U-jre_x64_linux_hotspot_17.0.7_7.tar.gz
+RUN tar xvf jre17.tar.gz && rm jre17.tar.gz
+
+# Download the JDK and maven and add them to path for building OTP
+RUN curl -Lo jdk17.tar.gz https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.7%2B7/OpenJDK17U-jdk_x64_linux_hotspot_17.0.7_7.tar.gz
+RUN tar xvf jdk17.tar.gz && rm jdk17.tar.gz
+ENV JAVA_HOME=/java/jdk-17.0.7+7/
+ENV PATH="$JAVA_HOME/bin:$PATH"
```

Comment:
question(non-blocking): is it possible to install this via apt?

Reply:
https://github.com/mbta/OpenTripPlanner/pull/53#discussion_r1189044188

Lev and I talked about this (but with mvn) in the very similar PR for dockerizing in the other repo. It basically boils down to this being the recommended install method for both java and maven, and it allows us to define a specific version that's more up to date than whichever one the apt repo has decided is stable.